src/core_plugins/kibana/server/lib/export/collect_panels.js

```diff
@@ -14,10 +14,11 @@ export async function collectPanels(savedObjectsClient, dashboard) {
 
   if (panels.length === 0) return [].concat([dashboard]);
 
-  const docs = await savedObjectsClient.bulkGet(panels);
+  const { saved_objects: savedObjects } = await savedObjectsClient.bulkGet(panels);
```

Comment:
There's no difference from what you've done here, but we're doing a lot of bulk gets (starting at `collectDashboards`) through here, and inside of `collectIndexPatterns` and `collectionSearchSources`, which could _possibly_ be done as a combined request at least at one tier of this onion? I commented _here_ because I think it can happen here and we could do a single `_mget` rather than two.

Mind making an issue if you agree? Any _set_ of requests that we can reduce will reduce latency and load time.